kong.conf.default

```diff
@@ -874,6 +874,13 @@
                                  # If set to 0 (default), such cached entities
                                  # or misses never expire.
 
+
+#db_cache_neg_ttl = 0            # Time-to-live (in seconds) of database misses
+                                 # where no entity is returned and nil is cached.
+                                 #
+                                 # If not set, the value of db_cache_ttl will be
+                                 # used.
```

Comment:
Let's update the description to be more accurate, more detailed, and better aligned with its `db_cache_ttl` counterpart:

```
Time-to-live (in seconds) of a datastore miss (no entity).

If not specified (default), the `db_cache_ttl` value will be used instead.

If set to 0, datastore misses will never expire.
```